packages/satusehat/src/Adaptor.js

```diff
@@ -57,6 +57,7 @@ export function get(path, params = {}, callback = s => s) {
 
       return prepareNextState(state, response, callback);
     } catch (e) {
+      console.log({ error: e.body });
```

Comment:
I think `console.error(e)` would work better here (and in other cases) and be more robust